### ISSUE TITLE
dokka-cli

### ISSUE THREAD

**Issue by vampire184** [Kotlin/dokka#2753]
**Question**
Hi, it's me again. I have two doubts about the usage of dokka-cli:
 1. How to specify the output format as Markdown. 
 2. Can the configuration parameters of sourceRoots use relative paths? after repeated testing, I found that I can only use absolute paths.

**Screenshots**
![image](https://user-images.githubusercontent.com/6406079/203888774-5e3b810d-c178-4b37-b02f-490f591c7b47.png)

**Installation**
- Operating system: macOS
- Build tool: Gradle v7.4.2
- Dokka version: 1.7.20

**Comment by IgnatBeresnev** [comment]
Hi! Dokka's revised documentation (#2728) is not out yet, but it covers this exact question. You can have a look at the preview: https://ignatberesnev.github.io/static/dokka/docs/kotlinlang/cli.html#other-output-formats

TLDR: you need to add gfm/jekyll plugin jars to `pluginsClasspath`.

> Can the configuration parameters of sourceRoots use relative paths?

Unfortunately, not at the moment - there's a separate bug for it with an explanation, see #2571
